app/controllers/tickets_controller.rb

```diff
@@ -177,11 +177,11 @@ def create
       @ticket = Ticket.new(ticket_params)
     end
 
-    if @tenant.ticket_creation_is_open_to_the_world == false &&
+    if !@tenant.ticket_creation_is_open_to_the_world? &&
           current_user.nil? && !using_hook
       render status: :forbidden, text: t(:access_denied)
-    elsif can_create_a_ticket(using_hook) &&
-        @ticket.save_with_label(params[:label])
+    elsif can_create_a_ticket(using_hook)
+      @ticket.save_with_label(params[:label]) unless @ticket.is_a?(Reply)
```

Comment:
This seems wrong, first the if would not fire when the save_with_label returned false. Now it still will. It should be something like this:
`elsif can_create_a_ticket(using_hook) && (@ticket.is_a?(Reply) || @ticket.save_with_label(params[:label]))`